src/tiledimage.js

```diff
@@ -778,6 +783,21 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
     },
 
     /**
+     * @returns {Boolean} whether the tiledImage can load its tiles even when it has zero opacity.
+     */
+    getPreload: function() {
+        return this._preload;
+    },
+
+    /**
+     * Set true to load even when hidden. Set false to block loading when hidden.
+     */
+    setPreload: function(preload) {
+        this._preload = new Boolean(preload).valueOf();
```

Comment:
Just looking now, and Travis has told me not to use the Boolean constructor. This is the cleanest way I know to handle Boolean objects and Boolean primitives in the same way. Here, both objects and primitives become primitives. 

Otherwise, we need to 
1) Drop support for Boolean Objects
2) Handle this in multiple lines

These six lines below would behave just like the one line above:
```
if (preload instanceof Boolean) {
    this._preload = preload.valueOf();
}
else {
    this._preload = !!preload
}
```